README.md

```diff
@@ -43,7 +43,7 @@ Use the styles for your component from a global `theme` object. To provide the t
 ```
 // basic usage
 import React from 'react'
-import { ThemeProvider } from '@marigold/system'
+import { ThemeProvider as MarigoldProvider } from '@marigold/system'
```

Comment:
thank you for your note :) 
MarigoldProvider extends the ThemeProvider and was moved to the components package. I will update this.

Reply:
Thank you for fixing!